.travis.yml

```diff
@@ -1,4 +1,3 @@
 language: node_js
 node_js:
-  - "6"
-  - "8"
+  - "10"
```

Comment:
do i need to add any other node versions here?

Reply:
hm maybe 12? most of our systems are on that version